modules/update-service-overview.adoc

```diff
@@ -4,7 +4,7 @@
 // * updating/updating-cluster-within-minor.adoc
 // * updating/updating-cluster-cli.adoc
 // * updating/updating-cluster-rhel-compute.adoc
-// * updating/updating-disconnected-cluster.adoc
+// * updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc
```

Comment:
I personally prefer the flexible gray areas "restricted network" hints at, but #48887 is moving the docs-consensus shorthand to "disconnected", so we probably want to stick with that for filenames.

Reply:
I went ahead and updated the actual content to use disconnected instead of restrcited. I'll apply these changes to non user-facing things like filenames, anchors, and other metadata in [OSDOCS-4890](https://issues.redhat.com/browse/OSDOCS-4890). This work entails making over 50 changes to the content, and if I do this before these current PRs are merged, these changes would have to be duplicated across 5 PRs. 

Unless anyone has any objection, I'll save this effort for when I tackle the linked ticket in a single PR.